### DISCUSSION TITLE
Hosted app not creating PR after "nuke and re-onboard"

### DISCUSSION THREAD

**Discussion by RothAndrew** [renovatebot/renovate#15982] [Request Help]
Mend Renovate hosted github app stopped reacting to new changes to our repo (defenseunicorns/zarf-package-software-factory). So we tried the "nuke and re-onboard", but the new onboarding PR is not being created. Requesting staff trigger manually according to [docs site](https://docs.renovatebot.com/getting-started/installing-onboarding/#nuke-config-and-re-onboard)

**Comment by rarkins** [comment]
Did it start working? I see multiple jobs, including from before your post:

<img width="736" alt="image" src="https://user-images.githubusercontent.com/6311784/172987111-f3373449-f934-4385-adbd-7f85b4149492.png">

**Reply by RothAndrew** [reply]
Yep! It just started working like 10 minutes ago.

**Reply by rarkins** [reply]
There was no manual intervention on our side - just coincidence. If you log into app.renovatebot.com you can access the above logs and perhaps learn why it wasn't "working" from your perspective previously in case it helps identify a bug or some other explanation
